### ISSUE TITLE
MSDN redirect for cs-cz content not working

### ISSUE THREAD

**Issue by nschonni** [MicrosoftDocs/feedback#617]
**Describe the bug**
I was searching for some old TFS client library documentation and found that search engines still have the CS-CZ content index. The bad search results/index is probably a separate issue.

**To Reproduce**
1. Go to https://duckduckgo.com/?q=Microsoft.TeamFoundation.VersionControl.Client.ItemType&ia=web
2. Click on the first MSDN link https://msdn.microsoft.com/cs-cz/library/microsoft.teamfoundation.versioncontrol.client.itemtype(v=vs.120).aspx
3. Note that you are on the old MSDN site and not redirected to docs.microsoft.com

**Expected behavior**
Should be redirected to https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/bb171043(v=vs.120)

**Screenshots**
There appears to be an additional language switcher on these pages
![image](https://user-images.githubusercontent.com/1297909/44291631-91607780-a24d-11e8-87e3-1c56637f3bb0.png)

**Desktop (please complete the following information):**
 - OS: Win 7
 - Browser Chrome
 - Version 68

**Additional context**
Removing the language code or replacing it with `en-us` redirects to the docs site. There is also a tranlated `cs-cz` version on the docs site.

**Comment by Powerhelmsman** [comment]
@sudeepku Is this your area?

**Comment by Powerhelmsman** [comment]
@nschonni Thanks for you feedback. This is known issue. Our team is still working on migration progress. Once it is done, the issue will be resolved. I will post you the status.
